packages/frontend/vite.config.js

```diff
@@ -0,0 +1,71 @@
+/**********************************************************************
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+/* eslint-env node */
+import { join } from 'path';
+import * as path from 'path';
+import { svelte } from '@sveltejs/vite-plugin-svelte';
+import { defineConfig } from 'vite';
+import { fileURLToPath } from 'url';
+
+let filename = fileURLToPath(import.meta.url);
+const PACKAGE_ROOT = path.dirname(filename);
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  mode: process.env.MODE,
+  root: PACKAGE_ROOT,
+  resolve: {
+    alias: {
+      '/@/': join(PACKAGE_ROOT, 'src') + '/',
+      '@shared/': join(PACKAGE_ROOT, '../shared') + '/',
```

Comment:
I think it should be /@shared and not @shared to not be confusing with global npmjs packages

```suggestion
      '/@shared/': join(PACKAGE_ROOT, '../shared') + '/',
```

Reply:
like https://www.npmjs.com/package/shared